src/Common/tests/System/Net/Http/Http2LoopbackConnection.cs

```diff
@@ -185,19 +186,28 @@ private async Task<Frame> ReadFrameAsync(CancellationToken cancellationToken)
             Stream oldStream = _connectionStream;
             _connectionSocket = null;
             _connectionStream = null;
-            _ignoreSettingsAck = false;
+            _ignoredSettingsAckPromise = null;
 
             return (oldSocket, oldStream);
         }
 
-        public void ExpectSettingsAck()
+        public async Task ExpectSettingsAckAsync(TimeSpan timeout)
```

Comment:
The code calling this method doesn't appear to need a customizeable timeout. It's reasonable to remove this param. Alternately, something like this would reduce the noise elsewhere:

```c#
public async Task ExpectSettingsAckAsync(int millisecondsTimeout = 30000)
```

Reply:
Also can we add a comment for this method? Calls it it look a lot like we're waiting for a settings ACK rather than just telling the connection that one will be coming in the future.

Reply:
It's reflecting the style already present in the class, e.g. https://github.com/dotnet/corefx/blob/c663f3ff12bcd95c54e9106aee218671d06fd675/src/Common/tests/System/Net/Http/Http2LoopbackConnection.cs#L115